backend/internal/storage/postgres/schema/review.go

```diff
@@ -0,0 +1,103 @@
+package user
+
+import (
+	"context"
+	"platnm/internal/models"
+
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type ReviewRepository struct {
+	db *pgxpool.Pool
+}
+
+func (r *ReviewRepository) GetReviews(ctx context.Context) ([]*models.Review, error) {
+	rows, err := r.db.Query(context.Background(), "SELECT user_id, media_id, media_type, desc, rating, CreatedAt, UpdatedAt FROM review")
+	if err != nil {
+		print(err.Error(), "from transactions err ")
+		return []*models.Review{}, err
+	}
+	defer rows.Close()
+
+	var reviews []*models.Review
+	for rows.Next() {
+		var review models.Review
+		var userID, mediaID, mediaType, desc, rating *string
+		var CreatedAt, UpdatedAt *time.Time
+
+		if err := rows.Scan(&userID, &mediaID, &mediaType, &desc, &rating, &UpdatedAt, &CreatedAt); err != nil {
+			print(err.Error(), "from transactions err ")
+			return reviews, err
+		}
+
+		review.UserID = *userID
+		review.MediaID = *mediaID
+		review.MediaType = *mediaType
+		review.Desc = desc
+		review.Rating = *rating
+		review.UpdatedAt = *UpdatedAt
+		review.CreatedAt = *CreatedAt
+
+		reviews = append(reviews, &review)
+	}
+
+	if err := rows.Err(); err != nil {
+		print(err.Error(), "from transactions err ")
```

Comment:
dont need log here. we added central error handling so if u just return error, fiber will catch it and log it